manifests/monitoring/kruize-monitoring-openshift.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: autotune
+  labels:
+    app: autotune
+  namespace: openshift-tuning
```

Comment:
Looks like this is assuming that the namespace exists. Need to add the namespace creation as well

Reply:
Added